docs/learn-accounts.md

```diff
@@ -97,6 +97,22 @@ and not officially affiliated with Web3 Foundation or the Polkadot project unles
 
 Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
 
+### Compatibility
+
+Seed phrases are not portable across all wallets. Some wallets only support certain signing curves
+and derivation paths. See the following table for compatibility.
+
+|                  | sr25519 |  ed25519  | secp256k1 |
+| :--------------- | :-----: | :-------: | :-------: |
+| Polkadot JS UI   |    X    |     X     |     X     |
+| Polkadot JS Apps |    X    |     X     |     X     |
+| Parity Signer    |    X    |    X\*    |           |
+| Ledger           |         | X&dagger; |           |
```

Comment:
Is this the Ledger app made by Zondax for Kusama? Or is it a generic ed25519 app on Ledger?

Reply:
Ledger app by Zondax currently only supports Ed25519+BIP32 hard derivations with `44'/434'` hardcoded prefix: https://github.com/Zondax/ledger-kusama/blob/master/app/src/coin.h#L27-L28.

They are planning to add sr25519, but AFAIU it will be covered by another round of grants by Web3 Foundation.